src/iinit.cpp

```diff
@@ -84,7 +301,11 @@ int main( int argc, char **argv )
     rErrMsg_t errMsg;
     rodsArguments_t myRodsArgs;
     int doPassword = 0;
-    bool doingEnvFileUpdate = false;
+
+    // THESE MUST BE DONE HERE! parseCmdLineOpt is EVIL and considers any unknown options invalid.
```

Comment:
maybe this comment should move down a couple lines?

Reply:
No, this comment applies to lines 307 and 308. If these options remain in `argv` when `parseCmdLineOpt` is called, an error will occur and `iinit` will terminate.